steampipecloud/utils.go

```diff
@@ -49,3 +49,20 @@ func randomString(n int) string {
 	}
 	return string(b)
 }
+
+func mapToJsonString(data map[string]interface{}) (string, error) {
```

Comment:
```suggestion
func mapToJSONString(data map[string]interface{}) (string, error) {
```

I think in Golang, acronym casing is usually preserved in function/variable names (unless it becomes unreadable) and it's consistent with the rest of the code